obspy/clients/filesystem/tests/test_sds.py

```diff
@@ -222,6 +222,48 @@ def test_sds_report(self):
             if failed:
                 raise Exception
 
+    def test_get_waveforms_bulk(self):
+        """
+        Test get_waveforms_bulk method.
+        """
+        for year, doy in ((2015, 213), (2015, 1), (2012, 1)):
```

Comment:
I think just testing one single time should be enough to ensure the bulk request works. I think I might have done it like this in some more basic tests, but this is really only about the bulk request.